content/en/serverless/installation/dotnet.md

```diff
@@ -32,6 +32,63 @@ Instrumenting .NET applications with the Datadog Lambda Extension is in beta.
 Datadog offers many different ways to enable instrumentation for your serverless applications. Choose a method below that best suits your needs.
 
 {{< tabs >}}
+{{% tab "Datadog CLI" %}}
+
+The Datadog CLI modifies existing Lambda functions' configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
+
+You can also add the command to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command _after_ your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
+
+### Install
+
+Install the Datadog CLI with NPM or Yarn:
+
+```sh
+# NPM
+npm install -g @datadog/datadog-ci
+
+# Yarn
+yarn global add @datadog/datadog-ci
+```
+
+### Configure credentials
+
+For a quick start, configure Datadog and [AWS credentials][1] using the following command. For production applications, consider supplying the environment variables or credentials in a more secure manner.
```

Comment:
I think the command is missing here?

Reply:
This is just us explaining how to set up the credentials for the terminal prior to providing the commands for isntrumenting the functions.

Reply:
Thanks, I updated the suggestion to say "instrumentation command" and linked it to the section.